src/pipecat/runner/types.py

```diff
@@ -65,3 +65,67 @@ class SmallWebRTCRunnerArguments(RunnerArguments):
     """
 
     webrtc_connection: Any
+
+
+@dataclass
+class SmallWebRTCPrebuiltArguments:
+    """Arguments for Small WebRTC prebuilt console template.
+
+    Parameters:
+        no_rtvi: Disables RTVI related functionality. Default: False
+        server_rtvi_version: Specifies the RTVI version in use by the server. Default: None
+        no_user_audio: Disables user audio input entirely. Default: False
+        no_user_video: Disables user video input entirely. Default: False
+        user_video_enabled: Enables user video input. Default: False
+        user_audio_enabled: Enables user audio input. Default: True
+        no_audio_output: Disables audio output for the bot. Default: False
+        no_bot_audio: Disables audio visualization for the bot. Default: False
+        no_bot_video: Disables video visualization for the bot. Default: True
```

Comment:
@jptaylor, the options feels a little bit confusing sometimes. Some options are phrased as negatives (`no_user_audio`) while others are phrased as positives (`user_audio_enabled`). 

It would be nice to make it clear regarting what is only for "visualization".

So, I would propose something like this for example:

```
user_audio_enabled: true     # User can speak
user_video_enabled: false    # User camera off by default
bot_audio_enabled: true      # Bot can output audio
bot_video_enabled: false     # Bot video output disabled
bot_audio_visuals_enabled: false  # No waveform or visualization
```

Reply:
And the same for the other options, instead of using no_something. Like `no_rtvi`, use it as `rtvi_enabled`.